vrecord

```diff
@@ -2179,11 +2231,13 @@ fi
 _get_decklink_inputs
 _get_avfoundation_inputs
 _get_avfctl_inputs
+_get_lang
+_set_lang_variables
```

Comment:
if these two functions are only relevant on Linux for the audio device then can you move then into the linux-conditional in _get_audio_device_list so that they don't unnecessarily run on other platforms.